romana/cmd/policy.go

```diff
@@ -136,7 +138,7 @@ func policyAdd(cmd *cli.Command, args []string) error {
 		}
 		fmt.Printf(util.JSONIndent(string(body)))
 	} else {
-		if resp.StatusCode == 201 {
+		if resp.StatusCode == http.StatusCreated {
```

Comment:
Wait! Do we post a Kubernetes policy definition to the Kubernetes API?

Our policy command is supposed to send a policy definition to the policy manager, which then turns around and sends this to the policy agents.

Not to Kubernetes.

We need this command in the case of OpenStack, where we don't properly integrate and we need our tools to set policies.

In the case of Kubernetes the user defines policies via the Kubernetes API and we pick this up along the way.